beacon-chain/core/electra/withdrawals.go

```diff
@@ -113,7 +113,8 @@ func ProcessWithdrawalRequests(ctx context.Context, st state.BeaconState, wrs []
 		}
 		validator, err := st.ValidatorAtIndexReadOnly(vIdx)
 		if err != nil {
-			return nil, err
+			log.WithError(err).Error("Could not get validator at index")
```

Comment:
if it exists in validator index by pubkey can it actually fail here?

Reply:
if it is needed we can provide some of the request information along with the log